Python: Adding missing integrations. (#7795)
getsentry/sentry-docs: src/platforms/python/common/configuration/integrations/boto3.mdx

---
title: Boto3
description: "Learn about the Boto3 integration and how it adds support for the Boto3 and botocore libraries."
sidebar_order: 26
---

> The Boto3 integration instruments requests made to Amazon Web Services done with [Boto3](https://boto3.amazonaws.com/v1/documentation/api/latest/index.html) (or the low level [botocore](https://github.com/boto/botocore) library that is used by Boto3 under the hood). For every request a span is created.

Reviewer: ```suggestion
The Boto3 integration instruments requests made to Amazon Web Services done with [Boto3](https://boto3.amazonaws.com/v1/documentation/api/latest/index.html) (or the low level [botocore](https://github.com/boto/botocore) library that Boto3 uses under the hood). It creates a span for every request.
```
